### PR TITLE
[ONNX] Modify softplus symbolic to support beta!=1

### LINKED ISSUE #64811: Onnx export of softplus with beta!=1
## 🐛 Bug

Onnx export of a softplus module with beta!=1 is currently not supported. Are there plans to  implement this in the future?

## To Reproduce
```python
import torch
beta, threshold = 13, 7
x=torch.linspace(-4,6,1000)
s=torch.nn.Softplus(beta=beta,threshold=threshold)
torch.onnx.export(s,x,'test.onx')
```
fails with 
```
 UserWarning: ONNX export failed on beta because has to be 1 not supported
  warnings.warn("ONNX export failed on " + op + " because " + msg + " not supported")
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-43-466f5e0cdaf3> in <module>
      4 x=torch.linspace(-4,6,1000)
      5 s=torch.nn.Softplus(beta=beta,threshold=threshold)
----> 6 torch.onnx.export(s,x,'test.onx')

lib/python3.8/site-packages/torch/onnx/__init__.py in export(model, args, f, export_params, verbose, training, input_names, output_names, aten, export_raw_ir, operator_export_type, opset_version, _retain_param_name, do_constant_folding, example_outputs, strip_doc_string, dynamic_axes, keep_initializers_as_inputs, custom_opsets, enable_onnx_checker, use_external_data_format)
    273 
    274     from torch.onnx import utils
--> 275     return utils.export(model, args, f, export_params, verbose, training,
    276                         input_names, output_names, aten, export_raw_ir,
    277                         operator_export_type, opset_version, _retain_param_name,

lib/python3.8/site-packages/torch/onnx/utils.py in export(model, args, f, export_params, verbose, training, input_names, output_names, aten, export_raw_ir, operator_export_type, opset_version, _retain_param_name, do_constant_folding, example_outputs, strip_doc_string, dynamic_axes, keep_initializers_as_inputs, custom_opsets, enable_onnx_checker, use_external_data_format)
     86         else:
     87             operator_export_type = OperatorExportTypes.ONNX
---> 88     _export(model, args, f, export_params, verbose, training, input_names, output_names,
     89             operator_export_type=operator_export_type, opset_version=opset_version,
     90             _retain_param_name=_retain_param_name, do_constant_folding=do_constant_folding,

lib/python3.8/site-packages/torch/onnx/utils.py in _export(model, args, f, export_params, verbose, training, input_names, output_names, operator_export_type, export_type, example_outputs, opset_version, _retain_param_name, do_constant_folding, strip_doc_string, dynamic_axes, keep_initializers_as_inputs, fixed_batch_size, custom_opsets, add_node_names, enable_onnx_checker, use_external_data_format, onnx_shape_inference)
    701 
    702             if export_params:
--> 703                 proto, export_map = graph._export_onnx(
    704                     params_dict, opset_version, dynamic_axes, defer_weight_export,
    705                     operator_export_type, strip_doc_string, val_keep_init_as_ip, custom_opsets,

RuntimeError: ONNX export failed: Couldn't export operator aten::softplus

Graph we tried to export:
graph(%0 : Float(1000, strides=[1], requires_grad=0, device=cpu)):
  %1 : Long(requires_grad=0, device=cpu) = onnx::Constant[value={13}]()
  %2 : Long(requires_grad=0, device=cpu) = onnx::Constant[value={7}]()
  %3 : Float(1000, strides=[1], requires_grad=0, device=cpu) = aten::softplus(%0, %1, %2) # 
lib/python3.8/site-packages/torch/nn/modules/activation.py:826:0
  return (%3)
```

## Expected behavior
Working export. What currently works (as a workaround) is to manually do the beta scaling
```python
from torch import Tensor
from torch.nn import functional as F

class Softplus2(torch.nn.Softplus):
      def forward(self, input: Tensor) -> Tensor:
        return F.softplus(input*self.beta, 1, self.threshold)/self.beta
    
s2=Softplus2(beta=beta,threshold=threshold)

y=s(x)
y2=s2(x)
torch.testing.assert_allclose(y,y2)

torch.onnx.export(s2,x,'test2.onx')
```
## Environment
```
PyTorch version: 1.9.0
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: Ubuntu 20.04.2 LTS (x86_64)
GCC version: (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
Clang version: Could not collect
CMake version: Could not collect
Libc version: glibc-2.31

Python version: 3.8.0 (default, Nov  6 2019, 21:49:08)  [GCC 7.3.0] (64-bit runtime)
Python platform: Linux-5.4.72-microsoft-standard-WSL2-x86_64-with-glibc2.10
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.20.2
[pip3] pytorch-lightning==1.3.8
[pip3] torch==1.9.0
[pip3] torchmetrics==0.4.1
[pip3] torchvision==0.10.0
[conda] _pytorch_select           0.1                       cpu_0  
[conda] blas                      1.0                         mkl  
[conda] libmklml                  2019.0.5                      0  
[conda] mkl                       2020.2                      256  
[conda] mkl-service               2.3.0            py38he904b0f_0  
[conda] mkl_fft                   1.3.0            py38h54f3939_0  
[conda] mkl_random                1.1.1            py38h0573a6f_0  
[conda] mypy_extensions           0.4.3                    py38_0  
[conda] numpy                     1.19.2           py38h54aff64_0  
[conda] numpy-base                1.19.2           py38hfa32c7d_0  
[conda] pytorch                   1.7.1           cpu_py38h6a09485_0
```

cc @BowenBao @neginraoof

### PR DESCRIPTION
**Pull request by shubhambhokare1**
Fixes #64811

### CODE REVIEW

File: torch/onnx/symbolic_opset9.py [hunk 1]
```diff
@@ -837,7 +837,8 @@ def softmax(g, input, dim, dtype=None):
 @parse_args("v", "t", "v")
 def softplus(g, self, beta, threshold):
     if beta != 1:
```
Comment (hwangdeyu) [inline torch/onnx/symbolic_opset9.py:839]: nit: we don't need `if beta != 1` condition after this changed right?

File: torch/onnx/symbolic_opset9.py [hunk 2]
```diff
@@ -837,7 +837,8 @@ def softmax(g, input, dim, dtype=None):
...
-        return _unimplemented("beta", "has to be 1")
+        beta = g.op("Constant", value_t=torch.LongTensor([beta]))
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset9.py:840]: Can `beta` be floating number?

Comment (BowenBao) [reply torch/onnx/symbolic_opset9.py:840]: nit: can remove the above parse_args, and use `_maybe_get_const` to check if beta == 1, then use original beta value as input in the below ops. By that you won't need to recreate Constant node for beta.

**Review by BowenBao — COMMENTED** [review]
@shubhambhokare1 thanks for PR, minor comments.

File: torch/onnx/symbolic_opset9.py (hunk 1 above)
Comment (shubhambhokare1) [reply torch/onnx/symbolic_opset9.py:839]: Having this check will skip some nodes that are not needed

File: torch/onnx/symbolic_opset9.py (hunk 2 above)
Comment (shubhambhokare1) [reply torch/onnx/symbolic_opset9.py:840]: `beta` can be a floating number, modified symbolic accordingly

**Review by BowenBao — APPROVED** [review]
LGTM, thanks!

**Pull request merged**